app/views/judgements/_form.html.erb

```diff
@@ -443,18 +461,27 @@
           Buttons
         </td>
         <% @rating_buttons.each do |b| %>
+          <%
+            @button_classes = "btn btn-default "
+            if @existing_rating == b["value"].to_i
+              @button_classes = @button_classes + "btn-preselected "
+            end
+          %>
           <td style="padding:5px;">
-            <%= button_tag b["label"], type: 'button', onclick: "rate("+b["value"]+")", class: 'btn btn-default '+b["class"], id:"button-for-"+b["key"][:name].to_s %>
+            <%= button_tag b["label"], type: 'button', onclick: "rate("+b["value"]+")", class: @button_classes + b["class"], id:"button-for-"+b["key"][:name].to_s %>
           </td>
         <% end %>
       </tr>
-
       <tr>
         <td style="padding:5px;text-align: center; color: #ccc;">
           Keyboard shortcuts
         </td>
-        <% @rating_buttons.each do |b| %>
-          <td style="padding:5px;text-align: center; color: #999;"><span id=<%= "span-for-" + b["key"][:name].to_s %>><%= b["key"][:display_name] %></span></td>
+        <% @rating_buttons.each do |b|
+            @span_style = ""
```

Comment:
can we use styles instead of explicit css here?

Reply:
will still do this

Reply:
obviously a quick thing, just missed it in my last few commits - will do this shortly